lightly/api/api_workflow_tags.py

```diff
@@ -73,3 +73,75 @@ def get_filenames_in_tag(
             masked_select_from_list(filenames_on_server)
 
         return filenames_tag
+
+    def create_tag_from_filenames(
+        self,
+        list_of_filenames: List[str],
```

Comment:
I would rename `list_of_filenames` to `filenames_new_tag` because 
- a) It is discouraged to use the type in the variable name
- b) It makes it clearer which filenames are meant.